lib/workers/repository/update/pr/changelog/source.ts

```diff
@@ -140,14 +140,12 @@ export abstract class ChangeLogSource {
           version,
           packageName,
           prev,
-          apiBaseUrl,
```

Comment:
are you sure? I thought they would be used when fetching changelogs from same platform but different host. 🤔

Reply:
The `endpoint` became unused when moving to `getPkgReleases`